kairos-test-utils/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "kairos-test-utils"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lib]
+
+[dependencies]
+
+tokio = { version = "*", features = [ "full", "tracing", "macros" ] }
+tempfile = "*"
+tracing = "0.1"
+nom = "*"
+
+[dev-dependencies]
+casper-client = "2"
+anyhow = "*"
```

Comment:
Instead of wildcards, I would rather see version specification.

Reply:
done https://github.com/cspr-rad/kairos/pull/34/commits/acd30355069f13f1194ec3468daaa9736656f00b